Wrappers/Python/ccpi/astra/processors/FBP.py

```diff
@@ -91,7 +102,7 @@ def __init__(self, volume_geometry,
                                       filter_type = filter_type)
             
             # Raise an error if the user select a filter other than ram-lak, for 2D case
-            if self.filter_type !='ram-lak':
+            if self.filter_type !='ram-lak' and self.device == 'cpu':
                 raise NotImplementedError('Currently in astra, 2D FBP is using only the ram-lak filter, switch to gpu for other filters')
```

Comment:
Please modify error message as follows: 'Currently with device=cpu, 2D FBP in ASTRA is only available with ram-lak filter, switch to gpu for other filters'